docs/documentation/stories/include-angular-material.md

```diff
@@ -9,22 +9,29 @@ ng new my-app
 cd my-app
 ```
 
-Install the `@angular/material` library and add the dependency to package.json...
+Install the `@angular/material` library and add the dependency to package.json.
 ```bash
 npm install --save @angular/material
 ```
 
-Import the Angular Material NgModule into your app module...
+Some Material components depend on the Angular animations module so install the `@angular/animations` library and add the dependency to package.json.
+```bash
+npm install --save @angular/animations
+```
+
+Import both the Angular Material and the BrowserAnimationsModule NgModules into your app module...
 ```javascript
 //in src/app/app.module.ts
 
 import { MaterialModule } from '@angular/material';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 // other imports
 
 @NgModule({
   imports: [
     ...
-    MaterialModule
+    MaterialModule,
+    BrowserAnimationsModule
```

Comment:
We should also inform users of NoopAnimationsModule if they prefer not to add another dependency